### DISCUSSION TITLE
Is it possible to sell multiple event tickets without specifying dates at once?

### DISCUSSION THREAD

**Discussion by jakobmeusburger** [HiEventsDev/Hi.Events#257] [Q&A]
The use case is the following: Our customer offers fitness courses and wants to sell "ticket passes". One ticket pass contains f.ex. 10 tickets. The customer can then come to any of the events and every time he attends his available tickets decrease by one. Would this workflow be possible with hi.events?

**Comment by daveearley** [comment]
This isn't a feature that's currently available. However, It would be nice to add. 

It could potentially be a "Number of redemptions allowed" setting on the ticket. Each time the ticket is scanned at check it could decrement this number.

**Reply by fantasybynumbers** [reply]
This is a feature we would love to have for the film festival we work with. It's what we currently use through another system and makes it so much easier for audience members to attend a collection of different screenings without having to pre-plan their entire schedule and juggle different tickets for different times.
